src/chat.js

```diff
@@ -0,0 +1,31 @@
+export function write_chat_msg(server, { json_msg, client }) {
+  for (const other_client of Object.values(server.clients)) {
+    other_client.write('chat', {
+      message: json_msg,
+      position: 0,
+      sender: client.uuid,
+    })
+  }
+}
```

Comment:
A code style i'd like to see for these kind of behavior is to write a mapper and then apply it. That way when we read the code we don't have to pay attention at the loop, the brain can focus on the behavior alone

```suggestion
export function write_chat_msg(server, { json_msg, client: { uuid } }) {
  const options = {
    message: json_msg,
    position: 0, // use an enum here, 0 is a magic value
    sender: uuid,
  }
  const send_packet = client => client.write('chat', options)
  Object.values(server.clients).forEach(send_packet)
}
```

Reply:
Can you rename json_msg into message ?